src/cljs/main/broadfcui/config.cljs

```diff
@@ -20,7 +20,7 @@
         optional {"cromwellVersion" :string "isDebug" :boolean "shibbolethUrlRoot" :string
                   "submissionStatusRefresh" :integer "userGuideUrl" :string
                   "workflowCountWarningThreshold" :integer "billingGuideUrl" :string
-                  "callCachingGuideUrl" :string}
+                  "callCachingGuideUrl" :string "alertsJsonUrl" :string}
```

Comment:
I don't even know why any of these are optional tbh, they all seem equally important